portal-frontend/src/app/features/public/application/submission/nfu-details/nfu-details.component.scss

```
@@ -1,7 +1,23 @@
+@use '../../../../../../styles/functions' as *;
+
 .soil-table {
   display: grid;
-  grid-template-columns: max-content max-content;
   overflow-x: auto;
-  grid-column-gap: 36px;
-  grid-row-gap: 12px;
+  grid-column-gap: rem(36);
+  grid-row-gap: rem(12);
+  grid-template-columns: max-content max-content max-content;
```

```suggestion
  grid-template-columns: max-content max-content;
```

Only two columns of data